packages/fxa-admin-server/src/subscriptions/test.util.ts

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const created = Date.now();
+export function addDays(current: number, days: number) {
```

Comment:
nit: I think a comment here might be helpful since I had to do a double take on what the code was doing and what the params were.